tests/lib/reset.sh

```diff
@@ -72,8 +70,15 @@ reset_classic() {
         # Restore snapd state and start systemd service units
         restore_snapd_state
         escaped_snap_mount_dir="$(systemd-escape --path "$SNAP_MOUNT_DIR")"
-        mounts="$(systemctl list-unit-files --full | grep "^${escaped_snap_mount_dir}[-.].*\\.mount" | cut -f1 -d ' ')"
-        services="$(systemctl list-unit-files --full | grep "^${escaped_snap_mount_dir}[-.].*\\.service" | cut -f1 -d ' ')"
+        all_units="$(systemctl list-unit-files --full | cut -f1 -d ' ')"
```

Comment:
This will include parts of the header and footer. You can fix this with a simple argument:
```suggestion
        all_units="$(systemctl list-unit-files --full --no-legend | cut -f1 -d ' ')"
```

Reply:
done